src/SDKs/ServiceBus/data-plane/README.md

```diff
@@ -51,4 +56,22 @@ In order to run the unit tests, you will need to do the following:
 
 Once you have completed the above, you can run `dotnet test` from the `/src/SDKs/ServiceBus/data-plane/tests/Microsoft.Azure.ServiceBus.Tests` directory.
 
-![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2FREADME.png)
\ No newline at end of file
+## Versioning information
+
+The Azure Service Bus client library uses [the semantic versioning scheme.](http://semver.org/)
+
+## Target frameworks
+
+For information about the target frameworks of the Azure Service Bus client library, please refer to the [Target Frameworks](https://github.com/azure/azure-sdk-for-net#target-frameworks) of the Microsoft Azure SDK for .NET.
+
+## Contributing
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+## Additional documentation
+
+- [Azure Service Bus General Documentation](https://docs.microsoft.com/en-us/azure/service-bus-messaging/)
+- [Azure Service Bus REST API Reference](https://docs.microsoft.com/en-us/rest/api/servicebus/)
+- [Azure Service Bus SDK for .NET Documentation](https://docs.microsoft.com/en-us/dotnet/api/overview/azure/service-bus?view=azure-dotnet)
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2Fsrc%2FSDKs%2FKeyVault%2Fdata-plane%2FREADME.png)
```

Comment:
This should point to the service bus impression, not keyvault, right?

Reply:
Ooops!   Good catch.   Fixed.